locations/tasks.py

```diff
@@ -0,0 +1,47 @@
+from future.standard_library import install_aliases
+install_aliases()  # noqa
+from urllib.parse import urlparse
+
+from celery.task import Task
+from django.conf import settings
+from seed_services_client import IdentityStoreApiClient
+
+from .models import Parish
+
+
+class SyncLocations(Task):
+    """
+    Has a look at all the identity store identities, and ensures that all of
+    the locations assigned to identities appear in the list of locations.
+    """
+    def get_identities(self, client):
+        """
+        Returns an iterator over all the identities in the identity store
+        specified by 'client'.
+        """
```

Comment:
Also, don't we only want to search through the identities that are healthcare workers (or VHTs or whatever)?